src/index.ts

```diff
@@ -8,10 +8,22 @@ import ConfigService from './config';
 import * as logging from './logging';
 import {renderServer} from './renderServer';
 import {renderStream} from './renderStream';
+import {PollingConfig} from './types';
 
 dotenv.config();
 Sentry.init({dsn: process.env.SENTRY_DSN});
 
+const defaultPollingConfig: PollingConfig = {
+  /**
+   * Poll ever 5 seconds on boot
+   */
+  bootInternval: 5 * 1000,
+  /**
+   * Poll ever 5min after the first config has been loaded
```

Comment:
5 minutes feels like quite a while to respond to changes, but I guess if we don't have backoff it's not bad

Reply:
Can definitely tweak this as we see fit down the line